src/components/Rules/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Typography, Table, TableBody, TableRow, TableCell, TableHead, Card, CardContent } from '@material-ui/core';
+
+interface RulesProps {
```

Comment:
why is this interface empty?

Reply:
thought I would need props when I was scaffolding; I'll remove it

Reply:
f7595c2a0c7354d091246e98611889f42fdc864b